### ISSUE TITLE
TesnsorDomain::allIDs() miss additional IDs introduced by setLoopDomain

### ISSUE THREAD

**Issue by naoyam** [NVIDIA/Fuser#3571]
Will add more info. Just a placeholder for now.

**Comment by kevinstephano** [comment]
The bug mentions this is a place holder.  Is this bug still relevant after all the RoPE work is finished @naoyam?

**Comment by naoyam** [comment]
Yes. It's more like a technical debt. Currently we work around this problem, but that limits the use cases of `setLoopDomain`, which in turn limits the flexibility of scheduling.
